### PR TITLE
Improve Android Google Signin Flow

### PR DESCRIPTION
**Pull request by yschimke**
- Use androidx.credentials

### CODE REVIEW

File: androidApp/src/main/java/dev/johnoreilly/confetti/account/SignInView.kt [hunk 1]
```diff
@@ -55,8 +56,27 @@ fun SignInRoute(component: SignInComponent) {
...
+        .build()
+
+    val result = credentialManager.getCredential(
+        context = context,
+        request = request
+    )
+
+    val credential = result.credential
+
+    if (credential is GoogleIdTokenCredential) {
```
Comment (yschimke) [inline androidApp/src/main/java/dev/johnoreilly/confetti/account/SignInView.kt:79]: TODO: error handling

**Comment by yschimke** [comment]
Old 
[signin_old.webm](https://github.com/joreilly/Confetti/assets/231923/25ed9942-8740-42de-83ad-4b09990e604f)

**Comment by yschimke** [comment]
New
[signi.webm](https://github.com/joreilly/Confetti/assets/231923/03ee9c65-7b19-423d-a2da-ef9ebed93fe0)

**Comment by joreilly** [comment]
nice!

**Comment by yschimke** [comment]
@joreilly any suggestions for how to pipe through the auth failures to a Snackbar?

**Comment by yschimke** [comment]
Now building ok.

**Pull request merged**